lib/common.js

```diff
@@ -341,13 +378,15 @@ exports.createTableForResources = function (resources) {
   } else {
     // the old way - show the dyno resources
     for (var name in resources.usage) {
-      var head = name + " (Max)";
-      if (name.toLowerCase() === "mem" || name.toLowerCase() === "disk") {
-        vals[head] = bytesToSize(resources.usage[name]) + " (" + bytesToSize(resources.max[name]) + ")";
-      } else if (name.toLowerCase() === "cpu") {
-        vals[head] = resources.usage[name] + "% (" + resources.max[name] + "%)";
-      } else {
-        vals[head] = resources.usage[name] + " (" + resources.max[name] + ")";
+      if (resources.usage.hasOwnProperty(name)) {
+        var head = name + " (Max)";
+        if (name.toLowerCase() === "mem" || name.toLowerCase() === "disk") {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ANestedIfDepth)